src/couch_mrview_http.erl

```diff
@@ -469,10 +469,16 @@ parse_params(Props, Keys, #mrargs{}=Args0, Options) ->
     IsDecoded = lists:member(decoded, Options),
     % group_level set to undefined to detect if explicitly set by user
     Args1 = Args0#mrargs{keys=Keys, group=undefined, group_level=undefined},
-    lists:foldl(fun({K, V}, Acc) ->
+    Args2 = lists:foldl(fun({K, V}, Acc) ->
         parse_param(K, V, Acc, IsDecoded)
-    end, Args1, Props).
-
+    end, Args1, Props),
+    Limit = Args2#mrargs.limit,
+    case config:get_integer("couch_db", " default_query_limit", -1) of
```

Comment:
`max_query_limit` confusing. Is it max value I can put into `limit` query parameter?